ui/src/components/policy/PolicyList.js

```diff
@@ -167,57 +268,93 @@ export default class PolicyList extends React.Component {
         this.setState({ showSuccess: null });
     }
 
+    inputChanged(key, evt) {
+        let value = '';
+        if (evt.target) {
+            value = evt.target.value;
+        } else {
+            value = evt ? evt : '';
+        }
+        this.setState({ [key]: value });
+    }
+
     render() {
-        const { domain, role } = this.props;
+        const { domain } = this.props;
         const left = 'left';
         const center = 'center';
-        const rows = this.state.list.map((item, i) => {
+        const policyNames = Object.keys(this.state.policiesMap);
+        const rows = policyNames.map((policyName, i) => {
+            let item = AppUtils.deepClone(
+                this.state.policiesMap[policyName][0]
+            );
+            let activeVersion = this.state.policiesMap[policyName].find(
+                (x) => x.active
+            );
+            // policy modified date of the active version
+            item.modified = activeVersion.modified;
+            // polciy row will be considered the active version (for assertions)
+            item.version = activeVersion.version;
             const name = NameUtils.getShortName(':policy.', item.name);
-            let color = '';
-            if (i % 2 === 0) {
-                color = colors.row;
-            }
             let onClickDeletePolicy = this.onClickDeletePolicy.bind(this, name);
+            let onClickDuplicatePolicyVersion =
+                this.onClickDuplicatePolicyVersion.bind(this, name);
             let newPolicy = name === this.state.successMessage;
             return (
                 <PolicyRow
                     name={name}
```

Comment:
Each row will show the active policy version (and expand button if there are multiple versions).
When clicking on expand, we'll attach the additional versions to the currently active version.